src/data/EXPLANATION.md

```diff
@@ -0,0 +1,11 @@
+### [Source](https://simplemaps.com/data/us-cities)
```

Comment:
```suggestion
# EXPLANATION

## [Source](https://simplemaps.com/data/us-cities)

```

Reply:
The first line of a markdown file needs to be a top level heading.

Headings need to be surrounded by blank lines.

You can install a markdown linting extension in VS Code, such as: https://marketplace.visualstudio.com/items?itemName=DavidAnson.vscode-markdownlint